src/hooks/index.ts

```diff
@@ -245,6 +248,39 @@ export const useTokenInfo = (
   return query.data
 }
 
+export const useNftModerationStatus = ({wallet, fingerprint}: {wallet: YoroiWallet; fingerprint: string | null}) => {
+  return useQuery({
+    queryKey: [wallet.id, 'nft', fingerprint],
+    queryFn: () => (fingerprint ? wallet.fetchNftModerationStatus(fingerprint) : null),
```

Comment:
when would fingerprint be undefined?

Reply:
When NFT is not found inside the `useNftImageModerated` hook